web/src/components/users/AdminUserEditForm.vue

```diff
@@ -0,0 +1,307 @@
+        />
+      </v-col>
+    </v-row>
+
+    <v-divider class="my-6"></v-divider>
+
+    <v-row>
+      <v-col
+        cols="12"
+        md="6"
+      >
+        <UserGroupAutocomplete
+          :model-value="groupMembershipAttributes.departmentId"
+          :type="UserGroupTypes.DEPARTMENT"
+          :parent-id="null"
+          :rules="[required]"
+          label="Department"
+          variant="outlined"
+          required
+          @update:model-value="updateDepartment"
+        />
+      </v-col>
+      <v-col
+        cols="12"
+        md="6"
+      >
+        <UserGroupAutocomplete
+          :model-value="groupMembershipAttributes.divisionId"
+          :type="UserGroupTypes.DIVISION"
+          :parent-id="groupMembershipAttributes.departmentId"
+          label="Division"
+          variant="outlined"
+          clearable
+          :readonly="isNil(groupMembershipAttributes.departmentId)"
+          :append-inner-icon="isNil(groupMembershipAttributes.departmentId) ? 'mdi-lock' : ''"
+          @update:model-value="updateDivision"
+        />
+      </v-col>
+    </v-row>
+
+    <v-row>
+      <v-col
+        cols="12"
+        md="6"
+      >
+        <UserGroupAutocomplete
+          :model-value="groupMembershipAttributes.branchId"
+          :type="UserGroupTypes.BRANCH"
+          :parent-id="groupMembershipAttributes.divisionId"
+          label="Branch"
+          variant="outlined"
+          clearable
+          :readonly="isNil(groupMembershipAttributes.divisionId)"
+          :append-inner-icon="isNil(groupMembershipAttributes.divisionId) ? 'mdi-lock' : ''"
+          @update:model-value="updateBranch"
+        />
+      </v-col>
+      <v-col
+        cols="12"
+        md="6"
+      >
+        <UserGroupAutocomplete
+          :model-value="groupMembershipAttributes.unitId"
+          :type="UserGroupTypes.UNIT"
+          :parent-id="groupMembershipAttributes.branchId"
+          label="Unit"
+          variant="outlined"
+          clearable
+          :readonly="isNil(groupMembershipAttributes.branchId)"
+          :append-inner-icon="isNil(groupMembershipAttributes.branchId) ? 'mdi-lock' : ''"
+          @update:model-value="updateUnit"
+        />
+      </v-col>
+    </v-row>
+
+    <v-row>
+      <v-col
+        cols="12"
+        md="6"
+      >
+        <h3 class="mb-1">Roles</h3>
+
+        <v-chip
+          v-for="(roleType, index) in user.roleTypes"
+          :key="index"
+          class="ma-2"
+          color="info"
+          size="large"
+        >
+          {{ formatRole(roleType) }}
+        </v-chip>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-spacer />
+      <v-col class="d-flex justify-end">
+        <v-btn
+          :loading="isLoading"
+          color="error"
+          variant="outlined"
+          v-bind="cancelButtonOptions"
+          text="Cancel"
+        />
+        <!-- TODO: add ability to request update of yukon government directory information during save -->
```

Comment:
Interestingly enough, I just built this in WRAP, and I wish I had put it in the "read" view rather than the "edit" view.
https://yg-wrap-uat.azurewebsites.net/profile
![image](https://github.com/user-attachments/assets/f81d9b65-f200-46a6-8b66-e18c992cba3c)
https://yg-wrap-uat.azurewebsites.net/profile/edit
![image](https://github.com/user-attachments/assets/b1c2538b-6fa3-40d8-a1c7-c596450dd656)

I probably should have put the "Sync with Directory" button where the "Edit" button is, and moved the "Edit" to the bottom right of the "read" view.